charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -1042,7 +1042,7 @@ def filesystem_mounted(fs):
 def make_filesystem(blk_device, fstype='ext4', timeout=10):
     """Make a new filesystem on the specified block device."""
     count = 0
-    e_noent = os.errno.ENOENT
+    e_noent = errno.ENOENT
```

Comment:
Curious, why is `errno` in global scope?

Reply:
i guess because there's a couple of places in this file where the module is used?